lib/qmp.ml

```diff
@@ -47,6 +47,8 @@ type fd_info = {
   fdset_id : int;
 }
 
+type qom = {name:string; ty:string;}
```

Comment:
Please use the same style as the declarations above

Reply:
updated